### PR TITLE
Pi5 runner

### PR DESCRIPTION
**Pull request by al3xa23**
Hi, 

just realized that the old PR from this branch was closed. I rebased this branch so that we can merge it.

### CODE REVIEW

**Review by stv0g — CHANGES_REQUESTED** [review]
We should include the Arm64 Docker image into the manifest here:
https://github.com/VILLASframework/node/blob/248bbf55c7228c1ab106e1efdc178d56cdc3a157/.gitlab-ci.yml#L216

It would also nice if we could stick with the convention and use lowercase tags to select the runners. `PI5` falls outside the convention here..

**Comment by al3xa23** [comment]
arm64 is inclued. But for some reason, the tag PI5 only works in capital letters.

**Comment by stv0g** [comment]
@al3xa23 The Runner configuration in GitLab was missing the correct (lowercase) tag. I have fixed this:

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -184,6 +184,12 @@ pkg:docker:
         PLATFORM: linux/amd64
         ARCH: x86_64
         TRIPLET: x86_64-linux-gnu
+        TAG: docker
+      - DISTRO: debian
+        PLATFORM: linux/arm64/v8
+        ARCH: arm64
+        TRIPLET: aarch64-linux-gnu
+        TAG: PI5
```
Comment (stv0g) [inline .gitlab-ci.yml:192]:
```suggestion
        TAG: pi5
```

**Review by stv0g — APPROVED** [review]

**Pull request merged**